frontend/src/config/constants.js

```diff
@@ -0,0 +1,88 @@
+export const PATIENT_TERM = "person";
```

Comment:
the content of this file should be added to `frontend/src/config/constants.ts`

Reply:
Done.